pkg/engine/utils/exceptions.go

```diff
@@ -34,14 +34,14 @@ func MatchesException(
 			if polex.Spec.Conditions != nil {
 				passed, err := conditions.CheckAnyAllConditions(logger, policyContext.JSONContext(), *polex.Spec.Conditions)
 				if err != nil {
-					return nil
+					return nil, err
```

Comment:
this doesn't look correct to me

Reply:
You mean even if there is an error evaluating the condition, we continue looping through the other exceptions?

If yes, we should at least have a logger message so that users can know why exceptions didn't work.

Reply:
This brings up another fix. If there is some failure getting an exception, a rule response of type error is returned. It seems to me that we should have a logger message instead and apply the policy to the resource.

Reply:
Yup, one exception being erroneous should not prevent valid exceptions to apply IMO.